NEWgIT.Core.Tests/ForkFetcherServiceTests.cs

```diff
@@ -0,0 +1,19 @@
+namespace NEWgIT.Core.Tests;
+
+public class ForkFetcherServiceTests
+{
+    [Fact]
+    public async void ForkFetcherService_given_repository_owner_and_name_returns_name_of_forks()
+    {
+        var forks = await ForkFetcherService.Instance.FetchForks("itu-bdsa", "project-description");
```

Comment:
Maybe we should mock the api response instead of using an actaul repository. Probably fine though